tests/ibis/test_ibis_model.py

```diff
@@ -18,8 +21,8 @@ class BasicModel(DataFrameModel):
 def t_schema_basic():
     return DataFrameSchema(
         {
-            # "string_col": Column(str),
-            "int_col": Column(int),
+            # "string_col": Column(dt.String),
```

Comment:
is this commented on purpose?

Reply:
It's commented just because the string data type isn't implemented as of this PR; it's implemented (and this part is uncommented) in #1766